### ISSUE TITLE
MovieWriter in glNext

### ISSUE THREAD

**Issue by richardeakin** [cinder/Cinder#509]
So, was just peeking at MovieWriter and I'm not sure of the whole story yet, but it appears that the old QTKit impl won't work in glNext becasue its 32-bit only.  Also, while there doesn't seem to be an AVFoundation based impl yet in the cinder/glNext branch, I do see one in caleb's initial cinderblock [here](https://github.com/calebjohnston/Cinder-AvfImpl/blob/master/src/AvfWriter.mm).  Is it still just a TODO to integrate this?

Pinging @calebjohnston and @pizthewiz to see if they can shed any light on this.

**Comment by mtourne** [comment]
Didn't seem like @calebjohnston is working out of the box on glnext, at least the samples code isn't

So far I've been using OpenCV VideoWriter, which I believe will route things through QTKit with this combination of codec and filename.
It would be nice to be able to use AVFoundation as it's supposed to be more performant from what I read (leveraging the gpu ?).

This is the OpenCV code that works for me :
```
    cv::VideoWriter video_writer;
    int codec = CV_FOURCC('m', 'p', '4', 'v');
    int frame_rate = 30;
    int width = 800;
    int height = 600;
    bool is_color = true;

    video_writer.open("/tmp/video_output.mp4", codec,
                      frame_rate, cv::Size(width, height), is_color);
```

**Comment by andrewfb** [comment]
This should be addressed by #766
